src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
```

Comment:
it's better to rename this variable to lowercase, basically we use upper + snake keys for static variables that won't be changed

learn more about javascript naming convention - [link](https://www.robinwieruch.de/javascript-naming-conventions/)